pyvista/core/filters/data_set.py

```diff
@@ -2170,7 +2170,7 @@ def contour(  # type: ignore[misc]
         if isinstance(scalars, str):
             scalars_name = scalars
         elif isinstance(scalars, (Sequence, np.ndarray)):
-            scalars_name = 'Contour Data'
+            scalars_name = 'Contour Data'  # type: ignore[unreachable]
```

Comment:
scalars is typed `str | None` here. The docstring says array scalars are supported. Might just need to update the allowed type, e.g.   `str | VectorLike[float] | None` (assuming only 1D scalars are supported)